package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.5.3",
   "description": "Large-screen/TV support library for Enact, containing a variety of UI components.",
   "repository": "https://github.com/enactjs/sandstone",
```

Comment:
Should it have `type: git` or not?

Reply:
Even without type:git, repository information is displayed properly

Reply:
Would you please unify the format through the repos? If you want this, apply this format for all repos.

Reply:
I fixed it